ecosystem/sep-0024.md

```diff
@@ -663,6 +663,7 @@ Name | Type | Description
 `limit` | int | (optional) The response should contain at most `limit` transactions.
 `kind` | string | (optional) The kind of transaction that is desired. Should be either `deposit` or `withdrawal`.
 `paging_id` | string | (optional) The response should contain transactions starting prior to this ID (exclusive).
+`lang` | string | (optional) Defaults to `en`. Language code specified using [ISO 639-1](https://en.wikipedia.org/wiki/ISO_639-1). 
```

Comment:
@lijamie98 Thanks for opening this PR!

I tried pushing this new change to your branch but seems like I don't have permission. So I created this PR against your branch with a suggestion to update the `lang` definition in all places of Sep-24: https://github.com/stellar/stellar-protocol/pull/1192

Reply:
Thanks for pushing the PR to make all `lang` definition consistent. 👍  I had merged the #1192.

Reply:
@lijamie98 thanks Jamie, I think you also need to accept the 2 suggestions I've made to your original PR here

Reply:
Would you mind also adding the `lang` change to the Changelog at the bottom? Thanks!

Reply:
@lijamie98 sorry but I think the suggestion from this very comment has not been accepted(pushed) yet ☝️ (seems like only 1 of the suggestions has been accepted)

The PR change for line `666` is still using the old text: ``lang` | string | (optional) Defaults to `en`. Language code specified using [ISO 639-1](https://en.wikipedia.org/wiki/ISO_639-1).`

Reply:
it should read like this:
`lang` | string | (optional) Defaults to `en`. Language code specified using [RFC 4646] which means it can also accept locale in the format `en-US`.

Reply:
@lijamie98 line `666` is still not updated